regression/ansi-c/invalid_redeclaration1/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE test-c++-front-end
+main.c
+
+^EXIT=(64|1)$
+^SIGNAL=0$
+(function symbol 'a' redefined with a different type|symbol 'c' is unknown)
```

Comment:
FYI, gcc reports `error: a parameter list without types is only allowed in a function definition`.

Reply:
That will be true with more recent GCC versions (which have become stricter about the use of function declarations without parameters), but GCC 9 still only says "error: conflicting types for ‘a’". I don't think we want to become as strict as more recent GCC versions are to keep supporting old, non-standards-compliant code.